src/main/java/LacksDataBaseProject/AddAndRemoveUser.java

```diff
@@ -0,0 +1,57 @@
+package LacksDataBaseProject;
+
+import java.util.Scanner;
+
+public class AddAndRemoveUser extends CheckUserRole {
+    static Scanner input = new Scanner(System.in);
+    private String password2;
+
+    protected void addUser(User addingUser, User addedUser) {
+        if (checkIfAdmin(addingUser)) {
+            System.out.println("Name: ");
+            addedUser.userName = input.next();
+            System.out.println("SkypeID: ");
+            addedUser.skypeID = input.next();
+            System.out.println("Login: ");
+            addedUser.login = input.next();
+            System.out.println("Password: ");
+            addedUser.password = input.next();
+            System.out.println("Repeat password: ");
+            password2 = input.next();
+            if (addedUser.password.equals(password2)) {
+                while (true) {
+                    System.out.println("Select user roles: ");
+                    for (int i = 0; i < 2; i++) {
+                        System.out.println("\t" + (i + 1) + ". " + Role.values()[i]);
+                    }
+                    int addUserRole = input.nextInt();
+                    addedUser.setRole(Role.values()[addUserRole - 1]);
+                    addedUser.listOfUsers.add(addedUser);
+                    break;
+                }
+            } else {
+                System.out.println("Password is not correct.");
+            }
+        } else {
+            return;
+        }
+    }
+
+    protected void removeUser(User removingUser, User removedUser) {
+        if (checkIfAdmin(removingUser)) {
+            System.out.println("Give skype ID, to remove: ");
+            String skypeID = input.next();
+            for (User userToRemove : removedUser.listOfUsers)
+                if (userToRemove.getSkypeID().equals(skypeID) && userToRemove.getRole() != Role.ADMIN) {
+                    removingUser.listOfUsers.remove(userToRemove);
+                    System.out.println("User was removed.");
+                    break;
+                }
+        } else {
+            return;
```

Comment:
ten else jest raczej nie potrzebny tutaj, bo skoro nawet jeśli if się wykona, to potem nie ma żadnych dodatkowych instrukcji :)